taxonomies/ingredients.txt

```diff
@@ -44770,6 +44811,10 @@ fr:oignons frais
 it:cipolla fresca
 # ingredient/fresh-onions has 485 products in 3 languages @2018-12-06
 
+<en:onion
+en:Onion pieces
+de:Zwiebelstücke
+
```

Comment:
```suggestion
```
This is unneeded as `stücke` is properly handled by the processing taxonomy. See [test](https://world-de.openfoodfacts.org/cgi/test_ingredients_analysis.pl?ingredients_text=Zwiebelst%C3%BCcke&type=add&action=process&.submit=1)

Reply:
Okay, I think I saw it as not recognized on a product page but I am not sure.